lib/perl/Genome/Model/Tools/Analysis/DumpIgvXmlMulti.pm

```diff
@@ -97,7 +98,8 @@ XML
 
     my $i=0;    
     for($i=0;$i<@bams;$i++){
-        my $path = abs_path($bams[$i]);
+        my $uri = URI->new($bams[$i]);
+        my $path = $uri->scheme ? $bams[$i] : abs_path($bams[$i]);
```

Comment:
It looks like we're using an old version of URI (1.52) that doesn't support `has_recognized_scheme`.